src/core/ui.js

```diff
@@ -161,6 +176,10 @@ export const ui = {
     if (owner) {
       owner.setAttribute("aria-expanded", "false");
     }
+    respecPill.setAttribute("aria-expanded", String(false));
```

Comment:
The conversion here is redundant: 

```suggestion
    respecPill.setAttribute("aria-expanded", "false");
```